### PR TITLE
Pract. 06: Add 00-03 | Rami Hennawi

### PR DESCRIPTION
**Pull request by RamiHennawi**
Rami Hennawi
SI 1 3MI0600370

### CODE REVIEW

File: Practicum/Pract. Rami/pract 06/02/NumberSeries/NumberSeries.cpp [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+		}
+	}
+
+	return false;
+}
+
+void NumberSeries::setInitialValue(int newValue) {
+	initialValue = newValue;
+
+	free();
```
Comment (khatoni) [inline Practicum/Pract. Rami/pract 06/02/NumberSeries/NumberSeries.cpp:121]: bug, initial value set to 0 in free()

File: Practicum/Pract. Rami/pract 06/02/NumberSeries/NumberSeries.cpp [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	generatedNumbersCount = 0;
+}
+
+void NumberSeries::setNewTransitionFunction(int (*newFunction)(int)) {
+	transitionFunction = newFunction;
```
Comment (khatoni) [inline Practicum/Pract. Rami/pract 06/02/NumberSeries/NumberSeries.cpp:126]: free() missing if I already have generated some numbers with previous function.

File: Practicum/Pract. Rami/pract 06/03/MultiSet/MultiSet.h [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+constexpr size_t MAX_NUMBER_COUNT = 3;
+
+class MultiSet {
+private:
+	int maxNumber = 0;
+	int* elements = nullptr;
```
Comment (khatoni) [inline Practicum/Pract. Rami/pract 06/03/MultiSet/MultiSet.h:8]: why int

**Review by khatoni — CHANGES_REQUESTED** [review]
Multiset is not memory efficient

**Comment by GeorgiTerziev02** [comment]
Closed due to inactivity

**Pull request closed without merging**